src/fides/api/main.py

```diff
@@ -321,14 +320,10 @@ def read_other_paths(request: Request) -> Response:
     # check first if requested file exists (for frontend assets)
     path = request.path_params["catchall"]
 
-    # First search in the local (dev) build for for a matching route.
-    ui_file = match_route(get_local_file_map(), path)
+    # search for matching route in package
+    ui_file = match_route(get_package_file_map(), path)
 
-    # Next, search for a matching route in the packaged files.
-    if not ui_file:
-        ui_file = match_route(get_package_file_map(), path)
-
-    # Finally, try to find the exact path as a packaged file.
+    # if not, try to find the exact path as a packaged file.
```

Comment:
I actually don't remember why we'd hit this case vs the previous case?

Reply:
okay I get it—the previous line is for _routes_ and this line is for everything else (i.e. css, images, etc.). I'm updating the comment to reflect that